File: Victor/bucles2/factorial.c
#include <stdio.h>

int main()
{
int contador;
int numero;
int resultado;
contador=0;
numero=0;
resultado=0;

	do
	{
		printf("Introduce un número para calcular su factorial: ");
		scanf("%d", &numero);
		if(numero>1)
		{
			contador=numero;
			resultado=numero;
			while (contador<=numero && contador>1)
			{
				printf(" %d x ",contador);
				contador--;
				resultado=resultado*contador;
			}
			printf("1 = %d \n",resultado);
		}
		else
		{
			printf("El número debe ser mayor que 1\n");
		}
	}
	while(numero<=1);
}

File: Guille/factorial/src/main/java/factorial.java



/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author guille
 */

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
/**
 *
 * @author guille
 */

/**
 *
 * @author guille
 */
public class factorial {
     public static void main(String[] args) 
     {
     try{
    System.out.println ("Introduce un número para calcular su factorial");
    BufferedReader in = 
            new BufferedReader(new InputStreamReader(System.in));
            int i = Integer.parseInt(in.readLine());
    
 for(int contador=1 ;contador <= i;contador ++)
    {
        
      contador*(contador+1)=int f
      System.out.println (int f);
    }

      
   
    catch (IOException ex){
                System.out.println(ex);
            }
  }

}